package.json

```diff
@@ -91,20 +65,28 @@
     "uuid": "^7.0.3"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-transform-regenerator": "^7.12.1",
+    "@babel/preset-env": "^7.12.11",
     "@typechain/ethers-v4": "^1.0.1",
+    "@types/chai": "^4.2.14",
```

Comment:
I think this 2 packages are not needed now :)

Reply:
Thanks for noticing. I removed types/chai, but why we don't need @typechain/ethers-v4? Interfaces that it generates are used in lib

Reply:
Sorry, I was not clear, I ment chai and chai as promised types :)